.eslintrc.json

```diff
@@ -37,6 +40,234 @@
 		"eqeqeq": [
 			"error",
 			"always"
-		]
+		],
+		"array-bracket-newline": [
+			"error",
+			"consistent"
+		],
+		"array-bracket-spacing": [
+			"error",
+			"never"
+		],
+		"array-element-newline": [
+			"error",
+			"consistent"
+		],
+		"arrow-parens": [
+			"error",
+			"as-needed"
+		],
+		"arrow-spacing": [
+			"error",
+			{
+				"before": true,
+				"after": true
+			}
+		],
+		"block-spacing": [
+			"error",
+			"always"
+		],
+		"brace-style": "off",
+		"@typescript-eslint/brace-style": [
+			"error",
+			"allman",
+			{
+				"allowSingleLine": true
+			}
+		],
+		"comma-dangle": [
```

Comment:
We're probably gonna change this in a separate PR as the extensibility with commas is a benefit, I find. Unless you have justification for not changing this?

Reply:
I chose the setting because it was more consistent with the current style. I'll keep this as is for now and create a new PR fixing it after this PR is merged. There's 1559 instances of `Missing trailing comma` so that way we can keep this PR smaller.

Reply:
We'll keep this as-is then for this PR and if you could please open a follow-up PR once this is merged to fix that, we'll then merge the results as we agree on PR size.